os-virt.te

```diff
@@ -20,6 +21,14 @@ allow virtlogd_t systemd_logind_t:dbus send_msg;
 allow systemd_logind_t virtlogd_t:dbus send_msg;
 allow virtlogd_t systemd_logind_inhibit_var_run_t:fifo_file write;
 
+#============= virtlogd_t ==============
+# allow access to /var/lib/nova directories which are labled with container_file_t
+# This is required for https://issues.redhat.com//browse/OSPRH-960
+allow virtlogd_t container_file_t:dir { add_name getattr remove_name search write };
+allow virtlogd_t container_file_t:file { create open unlink };
+allow virtlogd_t virt_var_run_t:file { append getattr lock open read };
```

Comment:
Is this really necessary as part of this patch? I'm looking at the [commit that added the virtlogd_t type](https://github.com/fedora-selinux/selinux-policy/commit/a42f11f216d14edc8be46d58e856a6b4acd21e37) and it seems like virtlogd_t is only meant to access specific log files (virtlogd_var_run_t) on purpose, while this would give it more general access to directories and files within the generic virt_var_run_t context.

If it's needed, could the commit message be updated to explain why? At the moment, it only mentions /var/lib/nova but this seems unrelated.

Reply:
this might also be a sideefect of my chcon

Reply:
ya it was so i have remvoed this